ansible/vagrant/Vagrantfile.Win2022.Adopt

```diff
@@ -46,8 +46,8 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |v|
     v.gui = false
     v.memory = 8192
-    v.cpus = 2
-    v.customize ["modifyvm", :id, "--cpuexecutioncap", "60"]
+    v.cpus = 4
```

Comment:
Could we compromise on 3? ;-)
```suggestion
    v.cpus = 3
```

Reply:
Im going to revert this and raise an issue, as I think some more investigation is required into why the vagrant boxes keep being terminated on the remote host.. changing number of cpus and execution cap, arent making much difference.. the playbooks run ok, but the "Run Build" fails on windows consistently.